src/core/src/validation_util.cpp

```diff
@@ -2,15 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/validation_util.hpp"
+#include "openvino/core/validation_util.hpp"
```

Comment:
Consider to move this file into dev API and merge with current "validation_util.hpp" there.

It would be better to have only one file in dev API which would be included like "openvino/core/validation_util.hpp"
Can be done as separate PR.

Reply:
IMO it should be done in a separate PR to not confuse the review.